src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -417,6 +426,17 @@ namespace AppInstaller::Repository
                     details.TrustLevel = SourceTrustLevel::Trusted | SourceTrustLevel::StoreOrigin;
                     result.emplace_back(std::move(details));
                 }
+
+                if (IsMSStoreDefaultSourceEnabled())
+                {
+                    SourceDetailsInternal details;
+                    details.Name = s_Source_MSStoreDefault_Name;
+                    details.Type = Rest::RestSourceFactory::Type();
+                    details.Arg = s_Source_MSStoreDefault_Arg;
+                    details.Identifier = s_Source_MSStoreDefault_Identifier;
+                    details.TrustLevel = SourceTrustLevel::Trusted;
+                    result.emplace_back(std::move(details));

```

Comment:
This should all come from just calling `GetWellKnownSourceDetails` rather than being implemented twice.